tests/Proto.Actor.Tests/EventStreamTests.cs

```diff
@@ -48,5 +49,18 @@ public void EventStream_OnlyReceiveSubscribedToEventTypes()
             eventStream.Publish("not an int");
             Assert.Equal(0, eventsReceived.Count);
         }
+
+        [Fact]
+        public void EventStream_CanSubscribeToSpecificEventTypes_Async()
+        {
+            var received = "";
+            var eventStream = new EventStream();
+            eventStream.Subscribe<string>(theString =>
+            {
+                received = theString;
+                Assert.Equal("hello", received);
```

Comment:
It does get called, well it did when I was debugging.

Obviously, the test will usually complete before as it's non-deterministic.  Wasn't sure how else to demo things.  By default, it does always use the `NullDispatcher`